.github/workflows/build.yml

```diff
@@ -65,7 +83,9 @@ jobs:
   setup-matrix:
     runs-on: ubuntu-latest
     outputs:
-      matrix: ${{ steps.setup-matrix.outputs.matrix }}
+      variants: ${{ steps.setup-matrix.outputs.variants }}
+      variants_arm64: ${{ steps.setup-matrix.outputs.variants_arm64 }}
+      arch: ${{ steps.setup-matrix.outputs.arch }}
     steps:
       - uses: actions/checkout@v3
```

Comment:
we can use `actions/checkout@v4`

Reply:
Done over at https://github.com/rstudio/r-docker/pull/132